sync_resources.py

```diff
@@ -1096,6 +1096,9 @@ def lambda_handler(req, context):
         if resource not in streams_to_invoke:
             logger.info(f"Skipping {resource} resource because this is a paginated invocation, and this resource is exhausted.")
             continue
+        if resource in excluded_resources:
```

Comment:
We could potentially skip resources during a paginated ingestion, correct? I imagine not a huge problem, but wanted to surface.

Reply:
I'm guessing you mean paginated "invocation"? And yeah I think if you're choosing to exclude it, it's fine.